firestore/src/main/firebase_metadata_provider_desktop.h

```diff
@@ -1,3 +1,5 @@
+// Copyright 2021 Google LLC
+
 #ifndef FIREBASE_FIRESTORE_CLIENT_CPP_SRC_MAIN_FIREBASE_METADATA_PROVIDER_DESKTOP_H_
 #define FIREBASE_FIRESTORE_CLIENT_CPP_SRC_MAIN_FIREBASE_METADATA_PROVIDER_DESKTOP_H_
```

Comment:
Lint: `Lines should be <= 80 characters long `<hidden value="cpplint-file-comment"></hidden>